presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -98,7 +98,7 @@ public static long currentTime(ConnectorSession session)
     {
         // Stack value is number of milliseconds from start of the current day,
         // but the start of the day is relative to the current time zone.
-        long millis = getChronology(session.getTimeZoneKey()).millisOfDay().get(session.getStartTime());
+        long millis = UTC_CHRONOLOGY.millisOfDay().get(session.getStartTime());
```

Comment:
This line has always raised concern and caused significant confusion.

Now, I finally realize that `current_time` has simply been completely wrong. And I realize that you also pointed this out in the document.

While I understand that Presto chose a very strange interpretation of time w/o tz, and timestamp w/o tz. It never occurred to me that existing `current_time` is simply completely wrong.

In addition, I also assumed the comment above is correct.

This fix deserves a lot more emphasis that it got. I hope you now understand why I'm asking all the questions and raising all the problems. It all comes from confusion as a result of reading this line.